File: examples/lh-server/go/02-workflows/workflow.go
package main

import "github.com/littlehorse-enterprises/littlehorse/sdk-go/littlehorse"

func QuickStart(wf *littlehorse.WorkflowThread) {
	name := wf.DeclareStr("name").Searchable()
	wf.Execute("greet", name)
}

File: examples/lh-server/python/02-workflows/workflow.py
from littlehorse.workflow import Workflow, WorkflowThread


def get_workflow() -> Workflow:
    def quickstart_workflow(wf: WorkflowThread) -> None:
        name = wf.declare_str("name").searchable()
        wf.execute("greet", name)

    return Workflow("quickstart", quickstart_workflow)
